src/main/resources/assets/appliedenergistics2/lang/en_US.lang

```diff
@@ -440,6 +439,11 @@ gui.tooltips.appliedenergistics2.AdvancedBlockingModeDefaultDesc=Blocking interf
 gui.tooltips.appliedenergistics2.AdvancedBlockingModeAll=Loose Mode
 gui.tooltips.appliedenergistics2.AdvancedBlockingModeAllDesc=Blocking interfaces will block on any item in this ME system.
 
+gui.tooltips.appliedenergistics2.CraftingModeStandard=Standard Crafting Mode
+gui.tooltips.appliedenergistics2.CraftingModeStandardDesc=Default Crafting Mode
```

Comment:
Replace with "Only starts craft if all ingredients are available in the ME System"